src/BugsnagLogger.php

```diff
@@ -37,8 +38,18 @@ public function __construct(Client $client)
         $this->client = $client;
     }
 
+    /**
+     * Set the notifyLevel of the logger, as defined in Psr\Log\LogLevel.
+     * 
+     * @param string $notifyLevel
+     * 
+     * @return void
+     */
     public function setNotifyLevel($notifyLevel)
     {
+        if (!in_array($notifyLevel, $this->getLogLevelOrder())) {
+            throw new TypeError("notifyLevel must be a valid Psr\Log\LogLevel value");
```

Comment:
We should be careful with hard failures, crashing the whole library. I think in some other places, we raise warnings, such as if the http layer fails.

Reply:
Also, `TypeError` doesn't exist in PHP 5, so can't be used here.

Reply:
I was going to say we should log the error and ignore the invalid input, but then we are within the logger 🤔 😄